rclrs/src/lib.rs

```diff
@@ -147,3 +152,136 @@ pub fn create_node(context: &Context, node_name: &str) -> Result<Node, RclrsErro
 pub fn create_node_builder(context: &Context, node_name: &str) -> NodeBuilder {
     Node::builder(context, node_name)
 }
+
+/// Extract non-ROS arguments from program's input arguments.
+///
+/// `args` is expected to be the input arguments of the program (passed via [`env::args()`]).
+/// According to rcl documentation, ROS arguments are between `--ros-args` and `--` arguments.
+/// Everything else is considered as non-ROS and left unparsed. Extracted non-ROS args are
+/// returned in the order that they appear (see example below).
+///
+/// # Example
+/// ```
+/// let input_args: Vec<String> = vec![
```

Comment:
In the other examples, I've used arrays since they're shorter and nowadays they also implement `IntoIterator`. But up to you.